ra/registration-authority.go

```diff
@@ -274,13 +289,13 @@ func (ra *RegistrationAuthorityImpl) validateContacts(ctx context.Context, conta
 				return problem
 			}
 			ra.stats.Inc("RA.ValidateEmail.Successes", 1, 1.0)
+			// continue for
```

Comment:
This comment is a little confusing. Let's restructure this loop to remove the switch, since there's really only two cases. We could do `if contact.Scheme != "mailto" { return error }`.